This change makes sure the IntelliJ IDEA headless instance used for code formatting does not store anything persistent
to the user's home directory, but to a directory in /tmp instead.

--- bin/idea.properties
+++ bin/idea.properties
@@ -5,12 +5,12 @@
 #---------------------------------------------------------------------
 # Uncomment this option if you want to customize path to IDE config folder. Make sure you're using forward slashes.
 #---------------------------------------------------------------------
-# idea.config.path=${user.home}/.IdeaIC/config
+idea.config.path=/tmp/IdeaIC/config

 #---------------------------------------------------------------------
 # Uncomment this option if you want to customize path to IDE system folder. Make sure you're using forward slashes.
 #---------------------------------------------------------------------
-# idea.system.path=${user.home}/.IdeaIC/system
+idea.system.path=/tmp/IdeaIC/system

 #---------------------------------------------------------------------
 # Uncomment this option if you want to customize path to user installed plugins folder. Make sure you're using forward slashes.
